src/queue.test.js

```diff
@@ -0,0 +1,10 @@
+import Queue from './queue';
+
+describe('Queue', () => {
```

Comment:
I really like the process of using test first.  are you going to use this today?

Reply:
thanks for merging my friend, nope I am not. I will ask more API/arch questions; feel free to use it if you like 😉